sources/models/src/modeled_types/shared.rs

```diff
@@ -1,9 +1,9 @@
+use super::error;
 use lazy_static::lazy_static;
 use regex::Regex;
+use semver::Version;
 use serde::{Deserialize, Deserializer, Serialize, Serializer};
 // Just need serde's Error in scope to get its trait methods
```

Comment:
Needing to import a trait is standard in Rust, renaming it to not conflict with another error also seems standard. I would just delete the comment so that the list of imports doesn't require further manual intervention.

```suggestion
```

Reply:
Yeah, fair enough. I'll just drop it.